docs/getting-started/verify.md

```diff
@@ -212,6 +212,39 @@ WARN[0000] you set --verify-image=false, image will not be verified
 RUNTIME.CONTAINERNAME  PID          UID          GID          MNTNS_ID RET FL… MODE        COMM        FNAME                  TIMESTAMP
 ```
 
+## Verify the ebpf-builder image
+
+We also sign the `ebpf-builder` image which is used to build gadgets.
+You can verify it using the following:
+
+```bash
+$ cosign verify --key inspektor-gadget.pub ghcr.io/inspektor-gadget/ebpf-builder:latest
+```
+
+We highly recommend you to verify by digest and specify the digest when building:
```

Comment:
```suggestion
We highly recommend you to verify by digest and specify the digest when building. This helps to protect against the TOCTOU problem. The image tag that got verified could point to another malicious image after the check.
```

I don't know if my explanation is worded good. But I think further information why verifying by digest is helpful